diff --git a/lib/loadSharp.js b/lib/loadSharp.js
index 1917738a3025944e96db81bb3c6981323f751859..66769f5f741d0b24f9d8a352e256c2c6ac5e8cca 100644
--- a/lib/loadSharp.js
+++ b/lib/loadSharp.js
@@ -1,6 +1,6 @@
 "use strict";
 
-const SHARP_RANGE = '^0.33.5';
+const SHARP_RANGE = '^0.35.4';
 
 // This is used to load sharp on the main thread, which prevents errors when worker threads exit
 // See https://sharp.pixelplumbing.com/install#worker-threads and https://github.com/lovell/sharp/issues/2263
diff --git a/package.json b/package.json
index c2eacfc7d3a02a76dcc2597c0aca9809e17ed49f..5b33159a0ef9992063d8eba9423afd6c9ec00832 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "nullthrows": "^1.1.1"
   },
   "devDependencies": {
-    "sharp": "^0.33.5"
+    "sharp": "^0.35.4"
   },
   "peerDependencies": {
     "@parcel/core": "^2.14.2"
diff --git a/src/loadSharp.js b/src/loadSharp.js
index 220bd7a4b8357a8dae164b4f3866859461667d30..792ef83e883a40657e4caf6bc14910f92ed0629d 100644
--- a/src/loadSharp.js
+++ b/src/loadSharp.js
@@ -2,7 +2,7 @@
 import type {PackageManager} from '@parcel/package-manager';
 import type {FilePath} from '@parcel/types';
 
-const SHARP_RANGE = '^0.33.5';
+const SHARP_RANGE = '^0.35.4';
 
 // This is used to load sharp on the main thread, which prevents errors when worker threads exit
 // See https://sharp.pixelplumbing.com/install#worker-threads and https://github.com/lovell/sharp/issues/2263
